pyproject.toml

```diff
@@ -101,8 +101,8 @@ multidict = "^4.6"
 aiohttp = "~3.6"
 questionary = "~1.5.1"
 prompt-toolkit = "^2.0"
-python-socketio = ">=4.4,<6"
-python-engineio = ">=3.11,<3.14"
+python-socketio = ">=5,<6"
+python-engineio = ">=4,<5"
```

Comment:
is there a specific reason we have to increase the minimum bound?

Reply:
Because of `>=4.4,<6`, `python-socketio` accidentally got bumped to `5`, which is incompatible with `python-engineio >=3.11,<3.14`. `python-socketio` and `python-engineio` need to be lockstep, as explained by rgstephens here: https://github.com/botfront/rasa-webchat/issues/344

Reply:
Sure, but can't poetry handle finding compatible versions if there exists a valid overlap?

Reply:
That would have been the hope, but there's no denying something went wrong [here](https://github.com/RasaHQ/rasa/pull/7479).

Reply:
For some reason, `poetry` seems to have a leaning for 5.0.0 for `python-socketio`. [Correct deps are defined from 5.0.1.](https://github.com/miguelgrinberg/python-socketio/commit/d2bb2b12e536036e34d94a1ad87e0d48a1a504a8). So there's another potential solution.

Reply:
ok I see 👍 
Would it be possible to just exclude `python-socketio 5.0.0`? 
We would like to be able to keep a wider range.